src/SourceGenerators/Uno.UI.SourceGenerators.Tests/XamlCodeGeneratorTests/Verifiers/CSGenerator.cs

```diff
@@ -16,6 +14,11 @@
 using Uno.UI.SourceGenerators.MetadataUpdates;
 using Uno.UI.SourceGenerators.XamlGenerator;
 
+#if !DEBUG && WRITE_EXPECTED
```

Comment:
I really prefer the previous version ... one is about being safe and tolerant in case of mistake (with a comment to explain to not do so), one is about blaming the user :/